packages/p2p/src/stores/my-profile-store.js

```diff
@@ -63,6 +63,14 @@ export default class MyProfileStore extends BaseStore {
         return list;
     }
 
+    @computed
+    get field_set() {
```

Comment:
`field_set` is too generic. Can you give a meaningful name?

Reply:
I will rename it as payment_method_field_set